ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -877,7 +1101,7 @@ org $00EEBB ; $006EBB
 db $A5, $8A, $C9, $1B, $00, $D0, $13, $A9
```

Comment:
nit: the `; Undo the function above:` comment appears 21 times and probably isn't necessary considering the flag comments explain how they enable/disable features

Reply:
fair